os/board/rtl8721csm/src/component/os/tizenrt/lwip_intf_tizenrt.c

```diff
@@ -285,6 +288,7 @@ unsigned char *rltk_wlan_get_ip(int idx)
 	
 	if (!dev_tmp) {
 		DiagPrintf("[rltk_wlan_get_ip] get dev fail\n");
+		return NULL;
```

Comment:
All failure case, please check free, even with line 306 and 321.
And who will free the memory for successful case?

Reply:
This one we do not need to free, we returned a global variable pointer instead.